src/activities/categorization/Categorization.qml

```diff
@@ -36,6 +36,7 @@ ActivityBase {
 
     property string boardsUrl: "qrc:/gcompris/src/activities/categorization/resource/"
     property bool vert: background.width < background.height
+    property string datasetURL: "../../GCompris-qt/src/activities/categorization/resource/board/"
```

Comment:
The qrc path was not working with opendir(). It returned NULL on using the qrc path.